src/encoded/static/components/item-pages/DefaultItemView.js

```diff
@@ -281,8 +282,9 @@ export default class DefaultItemView extends React.PureComponent {
      * @todo Maybe simplify CSS styling around these. Or get rid of these components and use plain HTML elements.
      */
     itemHeader(){
+        const { context, href, schemas, windoWidth } = this.props;
```

Comment:
Codacy found an issue: ['windoWidth' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5411404)